stable/mariadb/README.md

```diff
@@ -53,6 +53,7 @@ The following table lists the configurable parameters of the MariaDB chart and t
 | `image.pullPolicy`                        | MariaDB image pull policy                           | `Always` if `imageTag` is `latest`, else `IfNotPresent`           |
 | `image.pullSecrets`                       | Specify image pull secrets                          | `nil` (does not add image pull secrets to deployed pods)          |
 | `service.type`                            | Kubernetes service type                             | `ClusterIP`                                                       |
+| `service.clusterIp`                       | Specific cluster IP when service type is cluster IP. Use None for headless service |                                                                   |
```

Comment:
You need to indicate the default value or `nil` if the property is not set by default.

Reply:
Thanks juan131 , will do that.

Reply:
It is still pending, isn't it?